ercc/attestation/mock_ias.go

```diff
@@ -4,20 +4,13 @@ Copyright IBM Corp. All Rights Reserved.
 SPDX-License-Identifier: Apache-2.0
 */
 
-package mock
-
-import (
-	"github.com/hyperledger-labs/fabric-private-chaincode/ercc/attestation"
-	"github.com/hyperledger/fabric/common/flogging"
```

Comment:
Having some logging of the (top-level) attestation provider calls could be useful. That said, it seems we didn't have any before (but then i'm also surprised why go didn't complain about unused imports?)

Reply:
ehm the flogging import was indeed used by the `var logger`, however, the logger is no gone as this is not used.